src/airbrake.cr

```diff
@@ -49,22 +49,29 @@ module Airbrake
   class Config
     property :project_id
     property :project_key
+    property :host
+    property :port
+    property :secure
+
+    def uri
+      self.host   ||= "airbrake.io"
+      self.port   ||= 443
```

Comment:
The URI.new expects a separate param for the port. Also host and port are different so it would need some extra documentation. Having the separate property makes it more self documenting :)